plugins/modules/foreman_host.py

```diff
@@ -98,6 +98,21 @@
       - Mutually excluside with I(owner).
     type: str
     required: false
+  provision_method:
```

Comment:
And this is not available on host_groups?

Reply:
Correct